manager/src/test/scala/org/labrad/manager/ManagerConfigTest.scala

```diff
@@ -9,38 +9,39 @@ import scala.util.{Failure, Success}
 class ManagerConfigTest extends FunSuite {
 
   test("default manager config works with empty command line and env") {
-    val tryConfig = ManagerConfig.fromCommandLine(Array())
-    assert(tryConfig.isInstanceOf[Success[_]])
+    ManagerConfig.fromCommandLine(Array())
   }
 
   test("port can be set from environment variable") {
-    val Success(config) = ManagerConfig.fromCommandLine(Array(), env = Map("LABRADPORT" -> "7777"))
```

Comment:
yup, the LHS of an assignment can be an arbitrary pattern, just like in match cases. If the pattern doesn't match it will raise a MatchError, otherwise it unpacks and assigns just like in a match.